eda_analysis/eda_analysis.py

```diff
@@ -46,11 +49,43 @@ def describe_na_values(dataframe):
     numpy.ndarray
         A 2d Numpy Array of 1's and 0's, corresponding to the value of each entry in the dataframe.
         0 represents an NA value, 1 represents a non-NA value.
+    Examples:
+    ---------
+    >>> no_na_dataframe = pd.DataFrame({
+                                    "col_1": [0, 2],
+                                    "col_2": [0.5, 0.1],
+                                    "col_3": ["a", "b"]
+                                    })
+    >>> describe_na_variable(no_na_dataframe)
+    numpy.ndarray([1, 1],
+                  [1, 1],
+                  [1, 1])
+
+    >>>  na_numerical_dataframe = pd.DataFrame({
+                                           "col_1": [0, 2],
+                                           "col_2": [numpy.nan, 0.1],
+                                           "col_3": ["a", "b"]
+                                           }),
+    >>> describe_na_variable(na_numerical_dataframe)
+    numpy.ndarray([1, 1],
+                  [0, 1],
+                  [1, 1])
+    >>>  na_categorical_dataframe = pd.DataFrame({
+                                           "col_1": [0, 2],
+                                           "col_2": [0.5, 0.1],
+                                           "col_3": [np.nan, "b"]
+                                           }),
+    >>> describe_na_variable(na_numerical_dataframe)
+    numpy.ndarray([1, 1],
+                  [1, 1],
+                  [0, 1])
+
     '''
-    pass    
+    bool_array = dataframe.isna()
+    return np.array([[0 if val else 1 for val in bool_array[col]] for col in bool_array.columns])
```

Comment:
```suggestion
    na_val = np.array([[0 if val else 1 for val in bool_array[col]] for col in bool_array.columns])
    return pd.Series(data = np.sum(na_val!=1,axis=1),index = list(dataframe.columns))
```
OPTIONAL SUGGESTION:I really liked the one line logic you wrote for this function.But the array itself is not self explanatory. It is the same as finding the number of na values manually. You can return a series as suggested above. I tested that in my local by copying your function. You can use the suggested chunk or write your own logic,

Reply:
You can leave it for now as you will have to make changes in test file R function and its test. I am making it as an optional change